src/test/java/com/sryoondev/racingcar/step3/CarTest.java

```diff
@@ -4,37 +4,32 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
-import org.junit.jupiter.params.provider.ValueSource;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
 
 public class CarTest {
-    @Test
-    @DisplayName("자동차의 전진 테스트")
-    public void testMove() {
-        Car car = new Car();
-        assertThat(car.getMoveDistance()).isEqualTo(0);
-        car.move();
-        assertThat(car.getMoveDistance()).isEqualTo(1);
-    }
-
     @ParameterizedTest
     @DisplayName("입력 값에 따라 전진하거나 멈추는지 테스트")
     @CsvSource(value = {"true,1", "false,0"})
     void testRace(boolean movable, int moveDistance) {
-        Car car = new Car();
+        Car car = new Car("pobi");
         car.race(movable);
         assertThat(car.getMoveDistance()).isEqualTo(moveDistance);
     }
 
-    @ParameterizedTest
-    @DisplayName("이동거리 테스트")
-    @ValueSource(ints = {3, 10})
-    void testGetMoveDistance(int count) {
-        Car car = new Car();
-        for (int i = 0; i < count; i++) {
-            car.move();
-        }
-        assertThat(car.getMoveDistance()).isEqualTo(count);
+    @Test
+    @DisplayName("이름을 부여하는 차 객체 생성 테스트")
+    void testCarName() {
+        Car car = new Car("pobi");
+        assertThat(car);
```

Comment:
```suggestion
        assertThatCode(() -> new Car("pobi")).doesNotThrowAnyException();
```
정상인 코드를 테스트해보고 싶으실 땐, 위와 같은 방법을 사용해보세요 ^_^